src/napari/_vispy/utils/text.py

```diff
@@ -71,3 +73,168 @@ def _has_visible_text(layer: Points | Shapes) -> bool:
     ):
         return False
     return len(layer._indices_view) != 0
+
+
+def get_text_width_height(text: Text) -> tuple[float, float]:
+    """Get the screen space width and height of a vispy text visual."""
```

Comment:
Please improve the docstring to describe semantically what is returned (if it's pixels or other units).